exercises/rest-api/RestApi.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+public class User
+{
+    public string Name { get; }
+    public IDictionary<string, double> Owes { get; }
+    public IDictionary<string, double> Owed_By { get; }
+    public double Balance { get; set; }
+
+    public User(string name)
+    {
+        Name = name;
+        Owes = new Dictionary<string, double>();
+        Owed_By = new Dictionary<string, double>();
+    }
+}
+
+public class RestApi
+{
+    private IList<User> users;
```

Comment:
Could you remove this field? It is not necessary for the public test API and a user should be free to choose the underlying representation he/she chooses.